src/sidebar/test/media-embedder-test.js

```diff
@@ -90,6 +90,57 @@ describe('media-embedder', function () {
     });
   });
 
+  it('replaces mp3 links with html5 audio elements', function() {
+    var urls = [
```

Comment:
I would suggest combining these three tests into one slightly more generic "replaces audio links with html5 audio elements" test which just combines the URLs from the different tests.